### ISSUE TITLE
IP whitelist/blacklist

### ISSUE THREAD

**Issue by Ret2lib** [CTFd/CTFd#667]
It would be great to have the ability to block IP ranges or deny all but allow a few. Was initially thinking this would be a great feature for the cloud version since we don't get shell access to instance, but can be a neat feature overall. :)

**Comment by ColdHeat** [comment]
If you actively need this we can do whitelists on the managed instances. Just email support@ctfd.io to have it set up.

As for the open source version it should definitely be considered!

**Comment by ColdHeat** [comment]
It's also worth noting that this could easily be accomplished via a plugin 😄

**Comment by ColdHeat** [comment]
I don't think this is going to make it into 2.0 but it is a relatively easy plugin to implement.
